### PR TITLE
Fix #7739: Add 1.56 translations.

### LINKED ISSUE #7739: Add 1.56 translations

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #7739 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Test Plan:

Verify that the new playlist onboarding(https://github.com/brave/brave-ios/issues/7526) is correctly translated 

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 1]
```diff
@@ -562,15 +568,33 @@
...
+"customSearchEngine.customSearchEngineAddErrorTitle" = "Özel Arama Motoru Eklenirken Hata";
+
 /* The warning description shown to user when custom search engine will be deleted while it is default search engine. */
 "customSearchEngine.deleteEngineAlertDescription" = "Varsayılanken bir özel arama motorunu silmek, varsayılan motoru otomatik olarak değiştirecektir.";
 
 /* The alert title shown to user when custom search engine will be deleted while it is default search engine. The parameter will be replace with name of the search engine. */
 "customSearchEngine.deleteEngineAlertTitle" = "%@ arama motorunu silmek istediğinizden emin misiniz?";
 
+/* The warning description shown to user when custom search engine already exists. */
+"customSearchEngine.engineAlertExistsAlertDescription" = "Aynı adda bir arama motoru zaten var.";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "Aynı isimde bir arama motoru zaten var."

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 2]
```diff
@@ -562,15 +568,33 @@
...
+
+/* A description explaining that search template url is insecure, it is used for instance - Insecure Custom Search Template for Brave Search, Brave Search is a search engineand on a new seperate line */
+"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Şunun için Güvenli Olmayan Özel Arama Şablonu:";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "Güvenli Olmayan Özel Arama Şablonu:"

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 3]
```diff
@@ -562,15 +568,33 @@
...
+
+/* A description explaining that suggestion template url is insecure, it is used for instance - Insecure Custom Suggestion Template for Brave Search, Brave Search is name of search engine on a new seperate line */
+"customSearchEngine.insecureSuggestionTemplateURLErrorDescription" = "Şunun için Güvenli Olmayan Özel Öneri Şablonu:";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "Güvenli Olmayan Özel Öneri Şablonu:"

File: Sources/BraveStrings/Resources/zh.lproj/BraveShared.strings [hunk 4]
```diff
@@ -2593,6 +2620,24 @@
...
+"settings.highestQualityPlayback" = "最高质量播放";
+
+/* Detail text for the navigation link to quality settings view */
+"settings.highestQualityPlaybackDetail" = "支持高质量分辨率";
+
+/* Header for the general settings section */
+"settings.media.general" = "通用";
+
+/* Setting title for 'Media' sections. Media section offerts various options to manipulate media playback */
+"settings.media.rootSetting" = "媒體";
```
Comment (nuo-xu) [inline Sources/BraveStrings/Resources/zh.lproj/BraveShared.strings]:
```suggestion
"settings.media.rootSetting" = "媒体";
```

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 5]
```diff
@@ -3371,6 +3439,39 @@
...
+"vpn.autoRenewDiscountPopOverTitle" = "Brave VPN Aboneliğinizi hemen otomatik olarak yenileyin ve 3 ay boyunca %20 indirim kazanın!";
+
+/* Pop up title for renewing VPN subscription with month free */
+"vpn.autoRenewFreeMonthPopOverTitle" = "Brave VPN Aboneliğinizi hemen otomatik olarak yenileyin ve 1 ay ücretsiz kullanımdan faydalanın!";
+
+/* Pop up description for VPN subscription is about expire */
+"vpn.autoRenewSoonExpirePopOverDescription" = "Bu, Brave'in telefonunuzdaki tüm uygulamalar için sunduğu ekstra korumayı kaybedeceğiniz anlamına geliyor.";
+
+/* Pop up title for VPN subscription is about expire */
+"vpn.autoRenewSoonExpireTitle" = "Olamaz! Brave VPN aboneliğinizin süresi dolmak üzere.";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings:3473]: Made me chuckle but it is right translation for "Oh No!"

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 6]
```diff
@@ -3623,6 +3733,42 @@
 /* Whether the VPN plan has expired */
 "vpn.settingsVPNExpired" = "Süresi Dolmuş";
 
+/* Action button title that subscribe action for VPN purchase */
+"vpn.subscribeVPNActionButtonTitle" = "Hemen Abone Olun";
+
+/* Pop up description the subscription for VPN can be used for all platforms */
+"vpn.subscribeVPNAllDevicesPopOverDescription" = "Bu doğru. Brave VPN aboneliğiniz artık 5 cihaza kadar kullanılabilir. iOS üzerinden abone olup Mav, Windows ve Android cihazınızda ücretsiz kullanabilirsiniz.";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "Bu doğru. Brave VPN aboneliğiniz artık 5 cihaza kadar kullanılabilir. iOS üzerinden abone olup Mac, Windows ve Android cihazınızda ücretsiz kullanabilirsiniz."

Mac != Mav

**Review by nuo-xu — APPROVED** [review]

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 7]
```diff
@@ -578,13 +578,13 @@
 "customSearchEngine.deleteEngineAlertTitle" = "%@ arama motorunu silmek istediğinizden emin misiniz?";
 
 /* The warning description shown to user when custom search engine already exists. */
-"customSearchEngine.engineAlertExistsAlertDescription" = "Aynı adda bir arama motoru zaten var.";
+"customSearchEngine.engineAlertExistsAlertDescription" = "Aynı isimde bir arama motoru zaten var.";
 
 /* A description explaining that search template url is insecure, it is used for instance - Insecure Custom Search Template for Brave Search, Brave Search is a search engineand on a new seperate line */
-"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Şunun için Güvenli Olmayan Özel Arama Şablonu:";
+"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Güvenli Olmayan Özel Arama Şablonu:\" is enough no need for \"Şunun için";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: oh shoot my bad, this should be 

"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Güvenli Olmayan Özel Arama Şablonu:";

File: Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings [hunk 8]
```diff
@@ -578,13 +578,13 @@
...
 
 /* A description explaining that suggestion template url is insecure, it is used for instance - Insecure Custom Suggestion Template for Brave Search, Brave Search is name of search engine on a new seperate line */
-"customSearchEngine.insecureSuggestionTemplateURLErrorDescription" = "Şunun için Güvenli Olmayan Özel Öneri Şablonu:";
+"customSearchEngine.insecureSuggestionTemplateURLErrorDescription" = "Güvenli Olmayan Özel Öneri Şablonu:\" is enough no need for \"Şunun için";
```
Comment (soner-yuksel) [inline Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings]: "customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Güvenli Olmayan Öneri Şablonu:";

**Review by soner-yuksel — APPROVED** [review]

**Pull request merged**